go/cmd/dolt/commands/log.go

```diff
@@ -236,7 +259,11 @@ func logCommits(ctx context.Context, dEnv *env.DoltEnv, cs *doltdb.CommitSpec, o
 			return 1
 		}
 
-		commitsInfo = append(commitsInfo, logNode{commitMeta: meta, commitHash: cmHash, parentHashes: pHashes, branchNames: cHashToRefs[cmHash], isHead: cmHash == h})
+		commitsInfo = append(commitsInfo, logNode{commitMeta: meta,
```

Comment:
Kind of a nit but usually each field should go on its own line, including the first field